package exceptionHandling;

public class TestException {
public static void main(String [] args) {
	
	int a[] = new int[2];
	
	a[2]=10;
}
}